client/components/happychat/button.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import page from 'page';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import viewport from 'lib/viewport';
+import { openChat } from 'state/ui/happychat/actions';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+
+const HappychatButton = ( { translate, onOpenChat } ) => (
```

Comment:
That semicolon looks strange to me after `</Button>` but I'm all for removing unnecessary parenthesis.